packages/create-waku/src/index.ts

```diff
@@ -202,15 +204,40 @@ async function init() {
     await installTemplate(root, packageName, templateRoot, templateName);
   }
 
-  // TODO automatically installing dependencies
   // 1. check packageManager
   // 2. and then install dependencies
-
-  console.log(`\nDone. Now run:\n`);
-  console.log(`${bold(green(`cd ${targetDir}`))}`);
-  console.log(`${bold(green(commands.install))}`);
-  console.log(`${bold(green(commands.dev))}`);
   console.log();
+  console.log(`Installing dependencies by running ${commands.install}...`);
+
+  process.chdir(targetDir);
+
+  const installProcess = spawn(packageManager, ['install']);
```

Comment:
```suggestion
  const installProcess = spawn(packageManager, ['install'], { encoding: 'utf8' });
```

Reply:
Hi @himself65, `encoding` is not a property that exists on the options for `spawn`.

Reply:
first to know this

Reply:
Oh, I think if you just wanna bypass the output, use `stdio: 'inherit'`